### DISCUSSION TITLE
How to show time series in a web app.

### DISCUSSION THREAD

**Discussion by costantino78** [insarlab/MintPy#781] [Q&A] [answered]
Hi all.
Which library do you use to show the time series at this link?

https://mintpy.readthedocs.io/en/latest/hdfeos5/#4_web_viewer 

Is there any source code to view?

Thanks
Costantino

**Comment by yunjunz** [comment]
The web viewer is developed by @stackTom during his time at the University of Miami, he may know the answer to your question.

**Comment by stackTom** [comment, marked as answer]
> Hi all. Which library do you use to show the time series at this link?
> 
> https://mintpy.readthedocs.io/en/latest/hdfeos5/#4_web_viewer
> 
> Is there any source code to view?
> 
> Thanks Costantino

Sorry for the late reply. The source can be found here: https://github.com/geodesymiami/insarmaps

I plan to make the webapp available as a docker container to allow for hosting and viewing of files locally. Don't have an ETA for this though. Feel free to make a docker container and share it :).

**Comment by costantino78** [comment]
Thanks for the reply.

**Comment by stackTom** [comment]
There’s now a dockerfile provided which makes a docker container of the viewer. Instructions are in the readme, it’s straightforward.
